web/src/routes/(user)/people/+page.svelte

```diff
@@ -261,6 +268,37 @@
     changeName();
   };
 
+  const submitBirthDateChange = async (value: string) => {
+    showSetBirthDateModal = false;
+    if (!edittingPerson) {
+      return;
+    }
+    if (value === edittingPerson.name) {
+      return;
+    }
```

Comment:
These can be merged into a single condition.

Reply:
I also fixed the other one in submitNameChange (I was too fixated on following existing code eheh).